src/main/java/org/jboss/threads/Version.java

```diff
@@ -48,9 +51,17 @@ private Version() {
         }
         JAR_NAME = jarName;
         VERSION_STRING = versionString;
+
+        boolean logVersion = true;
         try {
-            Messages.msg.version(versionString);
+            logVersion = !"false".equalsIgnoreCase(System.getProperty(JBOSS_LOG_VERSIONS));
```

Comment:
I would strongly prefer usage of `parseBoolean` here, and use `"true"` as the default value for the property get.

Reply:
@dmlloyd yes let’s go with your PR.